src/commands/trace/__tests__/trace.test.ts

```diff
@@ -0,0 +1,54 @@
+// tslint:disable: no-string-literal no-null-keyword
+import {TraceCommand} from '../trace'
+
+describe('trace', () => {
+  describe('signalToNumber', () => {
+    test('should map undefined to undefined', () => {
+      process.env = {}
+      const write = jest.fn()
+      const command = new TraceCommand()
+      command.context = {stdout: {write}} as any
+
+      expect(command['signalToNumber'].call(undefined, null)).toBeUndefined()
+    })
```

Comment:
by using `.call` you're setting the `this` argument, but I don't think you need it in this case. The name of the test should also be `map null to undefined`. And you probably don't need the `.context`. We could do: 
```suggestion
    test('should map null to undefined', () => {
      const command = new TraceCommand()
      expect(command['signalToNumber'](null)).toBeUndefined()
    })
```